corehq/ex-submodules/couchexport/export.py

```diff
@@ -309,7 +309,8 @@ def split_path(path):
         id = []
         for k in path:
             if isinstance(k, basestring):
-                column.append(k)
+                if k:
+                    column.append(k)
```

Comment:
See 6c240e134c53507387bdcbf52a0cca743a2974d8 for what current functionality is prior to this change